python/sglang/srt/model_executor/model_runner.py

```diff
@@ -334,6 +334,21 @@ def init_memory_pool(
         max_num_reqs: int = None,
         max_total_tokens: int = None,
     ):
+        if self.server_args.kv_cache_dtype == "auto":
+            self.kv_cache_dtype = self.dtype
+        elif self.server_args.kv_cache_dtype == "fp8_e5m2":
+            if self.server_args.disable_flashinfer or self.server_args.enable_mla:
```

Comment:
Currently, only FlashInfer is supported and not Triton, due to the issue of insufficient smem. This needs to be fixed in another PR.